programs/ziti-edge-tunnel/package/rpm/postun.sh.in

```diff
@@ -5,5 +5,9 @@ systemctl daemon-reload >/dev/null 2>&1 || :
 %systemd_postun_with_restart ${SYSTEMD_SERVICE_NAME}
 
 if [ $1 -eq 0 ]; then
-    rm %{_unitdir}/${SYSTEMD_UNIT_FILE_NAME}
+    # delete the symlink to the executable that was created by the post scriptlet
+    unlink /usr/bin/$SYSTEMD_SERVICE_NAME
```

Comment:
This is the RPM remove scriptlet. We must delete the files we created in the install scriptlet because they're not specified in the package manifest, and therefore not automatically deleted during package removal.